tasks/fh-build.js

```diff
@@ -403,14 +382,43 @@ module.exports = function(grunt) {
         'shell:fh-run-array:integrate_cover', 'shell:fh-run-array:accept_cover',
         'shell:fh-report-cov:lcov', 'shell:fh-report-cov:cobertura'
       ]);
-    } else if (this.target === 'analysis') {
-      grunt.task.run(['plato:fh']);
-    } else if (this.target === 'shrinkwrap') {
-      grunt.task.run(['shell:fh-run-array:fhshrinkwrap']);
     } else if (this.target === 'default') {
-      grunt.task.run(['jshint', 'fh:test', 'fh:dist']);
+      grunt.task.run(['eslint', 'fh:test', 'fh:dist']);
     } else {
       grunt.fail.warn('Unknown target provided to `grunt fh`');
     }
   });
+
```

Comment:
We could probably replace the pre-existing `fh` multi-task with this rather than having both. I'm not sure if it's worth it right now though. I'd love to hear other thoughts on it!